misc/xml-to-record-converter/src/test/resources/xml/sample_28.xml

```diff
@@ -0,0 +1,7 @@
+<bk:book xmlns:bk="http://example.com/book" xmlns:au="http://example.com/author" xmlns:da="http://example.com/date">
+    <bk:title>string</bk:title>
+    <au:title>string</au:title>
+    <da:title>string</da:title>
+    <bk:chapter>1</bk:chapter>
+    <au:country>string</au:country>
+</bk:book>
```

Comment:
Shall we add test the ignore namespace cases for these newly added samples?

Reply:
We have `sample_31.bal` without namespaces.

Reply:
This is not similar to the `sample_31.bal`. Here for the `yy:title` the prefix(`yy`) is different for those all three `title` elements. Shall we add a test for this sample?

Expected type should be 
```
type Book record {
    int id;
    string[] title;
    int chapter;
    string country;
};
```